### ISSUE TITLE
[Feature Request] RTL Language Support

### ISSUE THREAD

**Issue by zigelnik** [BeforeIDieCode/BeforeIDieAchievements#152]
### Description

Unlike **English** and others, some languages are written from **Right-To-Left** (RTL), which can sometimes be challenging when translating.
On the main page, I noticed that it translated to my language which is in **Hebrew** but instead of putting the '**...**' after the sentence, it put it before.
I will now add a little adjustment that will include some of the RTL languages that were mentioned in #101.
Feel free to fix or add more in the future to support your own language.

## Possible Solution:
Create an array that will hold the wanted RTL language codes in such a way that it is easy to concatenate.
in `\src\Components\MasonryLayout\MasonryBox\MasonryBox.jsx` line 48' .

```
 const RTL_LANGUAGES = ["ar", "he", "az", "dv", "ku", "fa", "ur"];
  const BIDText = (
    <span className={styles.boldText}>
      {RTL_LANGUAGES.includes(languages[ipObj.ipInfo?.country_code]) ?
        <div className={styles["bid-text"]}> ...{t("BEFORE-I-DIE")} </div> :
        <div> {t("BEFORE-I-DIE")}... </div>
      }
      <br />
    </span>
  );
```

### Before changes:

![image](https://github.com/BeforeIDieCode/BeforeIDieAchievements/assets/113368830/bdbed442-bb7d-4b7f-93d5-f4ec5e4ffa3b)

### After changes:

![image](https://github.com/BeforeIDieCode/BeforeIDieAchievements/assets/113368830/f69faaf6-41a0-47dd-860a-37c9aaad7383)

### Checklist

- [X] I have checked for duplicate [issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
- [X] I have read the [Contribution Guidelines]((https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md))
- [X] I am willing to work on this issue (optional)

**Comment by XanderRubio** [comment]
@zigelnik Great job on the detailed and well-organized write-up about the open issue! The solution of using an array to hold RTL languages is a great UI fix that will benefit speakers of these languages. Thank you, Tal, for your contribution. Please proceed with creating a pull request✅

 I have a quick question, were the screenshots taken from your local server during development? I noticed that the map is not displayed under your name, and I just wanted to confirm if that is the case. As the code is using the Google Maps API and at first look at your screenshots this would be my guess as to why you are not able to see the contributor location on the map if it is in your local dev server.

**Comment by zigelnik** [comment]
> @zigelnik Great job on the detailed and well-organized write-up about the open issue! The solution of using an array to hold RTL languages is a great UI fix that will benefit speakers of these languages. Thank you, Tal, for your contribution. Please proceed with creating a pull request✅
> 
> I have a quick question, were the screenshots taken from your local server during development? I noticed that the map is not displayed under your name, and I just wanted to confirm if that is the case. As the code is using the Google Maps API and at first look at your screenshots this would be my guess as to why you are not able to see the contributor location on the map if it is in your local dev server.

Thanks again.
Yes, you are right. I did take those screenshots while running my local server run. Sharp eyes you have there.
I will now create a pull request and refer it to here. #154

**Comment by XanderRubio** [comment]
> > @zigelnik Great job on the detailed and well-organized write-up about the open issue! The solution of using an array to hold RTL languages is a great UI fix that will benefit speakers of these languages. Thank you, Tal, for your contribution. Please proceed with creating a pull request✅
> > I have a quick question, were the screenshots taken from your local server during development? I noticed that the map is not displayed under your name, and I just wanted to confirm if that is the case. As the code is using the Google Maps API and at first look at your screenshots this would be my guess as to why you are not able to see the contributor location on the map if it is in your local dev server.
> 
> Thanks again. Yes, you are right. I did take those screenshots while running my local server run. Sharp eyes you have there. I will now create a pull request and refer it to here. #154

Good to know, as I'm still always seeing what is the best way to develop and collaborate when we need to work on a feature that is using the API keys, and wanted to double-check 😉. This issue has been successfully closed from pull request #154. Thank you, Tal!
